inscriptions/ISic000479.xml

```diff
@@ -186,23 +191,38 @@
       <desc>Add Description Here</desc>
     </graphic>
   </surface>
-
     -->
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AR">
+                <ab>
+                    <lb n="1"/>memoriae
+                    <lb n="2"/> <name>Vegelae</name>
+                    <lb n="3"/>infelicissimae
+                    <lb n="4"/>quae <g ref="#interpunct">·</g>vixit <g ref="#interpunct">·</g>annis
+                    <lb n="5"/> <num value="21">XXI</num> <g ref="#interpunct">·</g> <expan><abbr>m</abbr><ex>ensibus</ex></expan> <g ref="#interpunct">·</g> <num value="3">III</num> <g ref="#interpunct">·</g> <expan><abbr>dieb</abbr><ex>us</ex></expan> <g ref="#interpunct">·</g> <num value="27">XVII</num> 
+                    <lb n="6"/> <name>Parthenopeus</name>
+                    <lb n="7"/>coniugi <g ref="#interpunct">·</g>dul 
+                    <lb n="8"break="no"/>cissimae 
+                </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app> 
+                        <note>CIL</note>
+                    </app>
+                </listApp>
+            </div>
+            <div type="translation" xml:lang="it" resp="#AR">
+                <p>Alla memoria della sventurata Vegela che visse ventuno anni tre mesi diciasette giorni. Partenopeo per l'amatissima coniuge</p>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#AR">
+                <p>In memory of the unfortunate Vegela who lived twenty one years three months and seventeen days. Parthenopeus for the beloved wife</p>
```

Comment:
I think in English we would assume 'for his beloved wife'. One might add the subaudible '(set this up)', but that is certainly optional. Also, just being super-picky (sorry!), 'twenty-one years, three months and...'